code/modules/overmap/exoplanets/exoplanet_flora.dm

```diff
@@ -44,7 +44,7 @@
 	S.set_trait(TRAIT_HIGHKPA_TOLERANCE,   atmosphere.return_pressure() + rand(5,50),500,110)
 	if(S.exude_gasses)
 		S.exude_gasses -= badgas
-	if(atmosphere)
+	if(atmosphere && length(atmosphere.gas))
```

Comment:
Nit: `length(atmosphere.gas)` would also null check it.

Reply:
wouldn't it runtime because null.gas?

Reply:
Oh I'm dumb and misread